pytest_timeout.py

```diff
@@ -223,9 +259,11 @@ def cancel():
 
         item.cancel_timeout = cancel
         timer.start()
+    return True
```

Comment:
why is this needed?

Reply:
`firstresult` hook processing stops on the first implementation that returns non-false-like.
The default `pytest_timeout_set_timer` implementation Is marked as `trylast=True` and should work without `return True` in the ideal world when nobody uses `trylast=True` for own hook overriding.
The explicit return:
1. makes the hook following the contract
2. generates less surprising errors with `trylast=True` sub-plugins: the subplugin is not called but the default implementation is used.
Bith reasons are not strong but I believe that dropping `return True` is a little worse.

Reply:
i see, thanks for explaining